Automattic/wp-calypso
# Move webpack jsx babel loader to calypso-builder

#### Changes proposed in this Pull Request

* Move the jsx transililation to the `calypso-builder`. 

#### Testing instructions

The usual:
```
   npm run distclean
   npm ci
   npm start
```

* Load calypso locally does it still work as expected.

Fixes https://github.com/Automattic/wp-calypso/issues/31370
